anydex/test/wallet/ethereum/test_eth_wallet.py

```diff
@@ -0,0 +1,190 @@
+        balance = {
+            'available': 992,
+            'pending': 0,
+            'currency': 'ETH',
+            'precision': 18
+        }
+        self.assertEqual(balance, await wallet.get_balance())
+
+    def test_get_outgoing_amount(self):
+        """
+        Test for the get_outgoing_amount function.
+        """
+        wallet = EthereumWallet(self.session_base_dir, True)  # Trick the wallet to not use the default provider
+        wallet.create_wallet()
+        wallet._session.add(
+            Transaction(is_pending=True, value=100, from_=wallet.account.address, hash=wallet.generate_txid()))
+        wallet._session.add(
+            Transaction(is_pending=True, value=200, from_=wallet.account.address, hash=wallet.generate_txid()))
+        self.assertEqual(300, wallet.get_outgoing_amount())
+
+    def test_get_incoming_amount(self):
+        """
+        Test for the get_incoming_amount function
+        """
+        wallet = EthereumWallet(self.session_base_dir, True)  # Trick the wallet to not use the default provider
+        wallet.create_wallet()
+        wallet._session.add(
+            Transaction(is_pending=True, value=100, to=wallet.account.address, hash=wallet.generate_txid()))
+        wallet._session.add(
+            Transaction(is_pending=True, value=200, to=wallet.account.address, hash=wallet.generate_txid()))
+        self.assertEqual(300, wallet.get_incoming_amount())
+
+    def test_get_address_not_created(self):
+        """
+        Test for getting the address of the wallet when it's not yet created
+        """
+        wallet = EthereumWallet(self.session_base_dir, True)  # Trick the wallet to not use the default provider
+        self.assertEqual("", wallet.get_address())
+
+    def test_get_address(self):
+        """
+        Test for getting the address of the wallet when it's created
+        """
+        wallet = EthereumWallet(self.session_base_dir, True)  # Trick the wallet to not use the default provider
+        wallet.create_wallet()
+        self.assertEqual(wallet.account.address, wallet.get_address())
+
+    def test_precision(self):
+        """
+        Test for the precision function
+        """
+        wallet = EthereumWallet(self.session_base_dir, True)  # Trick the wallet to not use the default provider
+        self.assertEqual(18, wallet.precision())
+
+    def test_get_identifier(self):
+        """
+        Test for get identifier
+        """
+
+        wallet = EthereumWallet(self.session_base_dir, True)  # Trick the wallet to not use the default provider
+        self.assertEqual("ETH", wallet.get_identifier())
+
+    async def test_get_transactions(self):
+        """
+        Test for get transactions
+        """
+
+        wallet = EthereumWallet(self.session_base_dir, True)  # Trick the wallet to not use the default provider
+        wallet.create_wallet()
+        tx = Transaction(hash="0x5c504ed432cb51138bcf09aa5e8a410dd4a1e204ef84bfed1be16dfba1b22060",
```

Comment:
This file contains a mix of single and double quotes. I think that should be changed to only single quotes.

Reply:
You are right, I will check my code and change everything i can find to single quotes.